### ISSUE TITLE
Knowledge distillation: A good teacher is patient and consistent, 2021

### ISSUE THREAD

**Issue by lisosia** [lisosia/cv_knowledge#22]
## 一言でいうと

BiT pretrained R152x2 を teacher にして R50 を学習して、ImageNetにてR50アーキテクチャで最高性能。
BiT pretrain R50以上の性能。

![image](https://user-images.githubusercontent.com/8558911/126345208-f461c6e3-9aa5-4caf-b09c-4c0373434525.png)

以下が重要
- teacher と student には augmentation 含めて同じ画像を入力して logit の一致を取る
- mixup を使う（ distillation というより、function matching する狙い、とのこと）
- とても長く学習する。Imagenetで4800epoch~

Adam > SGD らしい。
batchnorm -> groupnormに変更（teacher も group norm）

### 論文リンク
https://arxiv.org/pdf/2106.05237.pdf

### 著者/所属機関
Google Research

### 投稿日付(yyyy/MM/dd)
2021/6/9
